CODEOWNERS

```diff
@@ -49,8 +47,8 @@ src/math/*				@singalsu
 src/ipc/*				@xiulipan @bardliao
 src/drivers/intel/cavs/sue-ipc.c	@lyakh
 src/lib/*				@libinyang
-src/debug/gdb/*			@mrajwa
-src/schedule				@tlauda @mrajwa
+src/debug/gdb/*				@mrajwa
```

Comment:
It looks like some unintended whitespace change

Reply:
Diff, ugh. Tabs look weird in the diff, please check out locally, I would think it was misaligned before.

Reply:
Yea it was misaligned and github has problem with tabs ;p